examples/nlp/language_modeling/tuning/megatron_gpt_sft.py

```diff
@@ -202,7 +202,6 @@ def main(cfg) -> None:
             return_config=True,
             save_restore_connector=save_restore_connector,
         )
-        gpt_cfg = _modify_config(gpt_cfg, cfg, add_cfg_to_tree=False)
```

Comment:
Can you just delete this line?

Reply:
Yep. The reason is that in [the next line when](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/tuning/megatron_gpt_sft.py#L206) calling `load_from_nemo` we run exactly the same line again [here](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/tuning/megatron_gpt_sft.py#L96). So it's fine to delete, I tested that.